content/en/docs/releases/supported-releases/index.md

```diff
@@ -74,10 +71,11 @@ Please keep up-to-date and use a supported version.
 
 | Minor Releases   | Patched versions with no known CVEs                  |
 | ---------------- | ---------------------------------------------------- |
+| 1.18.x           | 1.18.0                                                |
 | 1.17.x           | 1.17.2+                                               |
 | 1.16.x           | 1.16.4+                                               |
 | 1.15.x           | 1.15.7 - End of life. A new CVE will NOT be patched  |
-| 1.14 and earlier | None, all versions have known vulnerabilities.       |
```

Comment:
I'd think about leaving this one in. At some point, the 1.15.7 won't have been updated and we can simply update this line to 1.15 and earlier and remove that one. Or just make that change know, removing the 1.15.x and changing the last to 1.15 and earlier.

Reply:
I'd suggest we put in 1.15. Just say It is EOL and users should upgrade ASAP. No need to specify if there are vulnerabilities or not.